disperser/dataapi/subgraph/queries.go

```diff
@@ -118,4 +121,7 @@ type (
 	queryOperatorEjectedsByOperatorID struct {
 		OperatorEjections []*OperatorEjection `graphql:"operatorEjecteds(orderBy: blockTimestamp, where: {and: [{blockTimestamp_gte: $blockTimestamp_gte}, {operatorId: $operatorId}]}, first: $first, skip: $skip)"`
 	}
+	queryReservations struct {
+		Reservations []*Reservation `graphql:"currentReservations(where: {startTimestamp_lte: $currentTimestamp, endTimestamp_gte: $currentTimestamp}, orderBy: startTimestamp, orderDirection: asc, first: $first, skip: $skip)"`
```

Comment:
```suggestion
	queryActiveReservations struct {
		Reservations []*Reservation `graphql:"currentReservations(where: {startTimestamp_lte: $currentTimestamp, endTimestamp_gte: $currentTimestamp}, orderBy: startTimestamp, orderDirection: asc, first: $first, skip: $skip)"`
```

Reply:
For consistency think we should use "queryActiveReservations" everywhere. I also commented on your other PR that "currentReservations" is unclear as it sounds like it means "activeReservations" but it actually just means "historicalReservations".

Reply:
TLDR: think we should change
- in other PR: currentReservations -> reservations (or allReservations)
- in this PR: queryReservations -> queryActiveReservations (because it filters for only those currently active)